Dockerfile

```
@@ -16,8 +17,24 @@ RUN apt-get update && apt-get install -y \
     # Media processing
     ffmpeg \
     # Utilities
-    curl unzip \
-    && rm -rf /var/lib/apt/lists/*
+    unzip
+
+SHELL ["bash", "-o", "pipefail", "-c"]
+
+RUN if [ "$(dpkg --print-architecture)" = "amd64" ]; then \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)